fe/fe-core/src/test/java/org/apache/doris/nereids/rules/rewrite/PullUpProjectUnderTopNTest.java

```diff
@@ -35,24 +36,27 @@
 
 class PullUpProjectUnderTopNTest implements MemoPatternMatchSupported {
     private final LogicalOlapScan scan1 = PlanConstructor.newLogicalOlapScan(0, "t1", 0);
+    private final LogicalOlapScan scan2 = PlanConstructor.newLogicalOlapScan(1, "t2", 0);
```

Comment:
why not add a new ut to cover the join with select alias, avoid overwrite the original one?